### ISSUE TITLE
cmake fortran with yaml fails ctest

### ISSUE THREAD

**Issue by scharlton2** [usgs-coupled/phreeqcrm#46]
```
$ cmake .. -DPHREEQCRM_BUILD_PYTHON:BOOL=ON -DPHREEQCRM_WITH_YAML_CPP:BOOL=ON -DCMAKE_BUILD_TYPE:STRING=Release -DPHREEQCRM_FORTRAN_TESTING:BOOL=ON
$ cmake --build . -j10
$ ctest
Test project /home/charlton/source/repos/coupled/phreeqcrm-master/_config/phreeqcrm-0.1.0-9067/_cmake
    Start 1: TestCompileAndRun
1/2 Test #1: TestCompileAndRun ................***Exception: SegFault  0.39 sec
    Start 2: SimpleAdvect.py
2/2 Test #2: SimpleAdvect.py ..................   Passed    0.05 sec

50% tests passed, 1 tests failed out of 2

Total Test time (real) =   0.44 sec

The following tests FAILED:
          1 - TestCompileAndRun (SEGFAULT)
Errors while running CTest
Output from these tests are in: /home/charlton/source/repos/coupled/phreeqcrm-master/_config/phreeqcrm-0.1.0-9067/_cmake/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
```

**Comment by scharlton2** [comment]
https://github.com/scharlton2/phreeqcrm/tree/default_ctor 
https://github.com/scharlton2/phreeqcrm/commit/78f53a3f00fe7d06ab6a306e086a27b561bf6ce9

**Comment by scharlton2** [comment]
Closed by PR #47
